src/App.js

```diff
@@ -6,6 +6,17 @@ import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
 
+import { PostList } from './PostList/PostList';
+
+const postsWithCommentsAndAuthors = posts.map(post => {
+  const currentPost = { ...post };
+
+  currentPost.commentsToPost = comments.filter(comment => comment.postId === currentPost.id);
```

Comment:
А навіщо уточнювати, якщо це об'єкт post то ясно що autor це автор саме цього посту, те саме з коментарями
```suggestion
  currentPost.comments = comments.filter(comment => comment.postId === currentPost.id);
```